### PR TITLE
R deps

### PR DESCRIPTION
**Pull request by druvus**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

More R dependencies for maaslin

### CODE REVIEW

File: recipes/r-gam/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+build:
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
```
Comment (bgruening) [inline recipes/r-gam/meta.yaml]: I think we agreed to use `r` as base dependency.

**Comment by bgruening** [comment]
@druvus if you are pain approved and have some time I would appreciate some help with this one: https://github.com/bioconda/bioconda-recipes/tree/r29

File: recipes/r-gam/meta.yaml (hunk 1 above)
Comment (druvus) [inline recipes/r-gam/meta.yaml]: thanks, I will replace it.

**Comment by druvus** [comment]
@bgruening I was looking into r29 but I am not sure what is going on there. I will look into it again during the week.

**Comment by bgruening** [comment]
Let's try this one :)

**Pull request merged**